SoftLayer/CLI/block/replication/failback.py

```diff
@@ -0,0 +1,23 @@
+"""Failback from a replicant volume."""
+# :license: MIT, see LICENSE for more details.
+
+import click
+import SoftLayer
+from SoftLayer.CLI import environment
+
+
+@click.command()
+@click.argument('volume-id')
+@click.option('--replicant-id', help="ID of the replicant volume")
+@environment.pass_env
+def cli(env, volume_id, replicant_id):
+    """Failback a block volume from the given replicant volume."""
+    block_storage_manager = SoftLayer.BlockStorageManager(env.client)
+
+    success = block_storage_manager.failback_from_replicant(
+        volume_id,
+        replicant_id
+    )
+
+    if success:
```

Comment:
If success is `False`, shouldn't we return some kind of error message to the user? I'm not sure what conditions `False` would be returned over returning an exception.